lib/setup-lib.bash

```diff
@@ -140,9 +140,9 @@ function direnv_shell_integration() {
     *bash*)
       rcfile="$HOME/.bashrc"
       # shellcheck disable=SC2016
-      asdf_direnv_rcfile_expr='${XDG_CONFIG_HOME:-$HOME/.config}/asdf-direnv/bashrc'
+      asdf_direnv_rcfile_expr='"${XDG_CONFIG_HOME:-$HOME/.config}/asdf-direnv/bashrc"'
       asdf_direnv_rcfile=$(eval echo "$asdf_direnv_rcfile_expr")
```

Comment:
Hm, haven't noticed we had `eval echo ...` in here.

@jfly do you think we can remove `asdf_direnv_rcfile_expr` ? and remove these evals (also on ~156) or am I missing something?